index.html

```diff
@@ -554,6 +554,775 @@ <h2>OpenID Connect</h2>
+SHOULD cease to be used. Once the value is set, it is not expected to be updated, and
+systems depending on the value are expected to not verify any proofs associated
+with the <a>verification method</a> at or after the time of revocation.
+                </dd>
+              </dl>
+            </dd>
+          </dl>
+
+          <pre class="example nohighlight"
+            title="Example verification method structure">
+    {
+      "@context": [
+        "https://www.w3.org/ns/did/v1",
+        "https://w3id.org/security/data-integrity/v1"
+      ]
+      "id": "did:example:123456789abcdefghi",
+      <span class="comment">...</span>
+      "verificationMethod": [{
+        "id": <span class="comment">...</span>,
+        "type": <span class="comment">...</span>,
+        "controller": <span class="comment">...</span>,
+        "publicKeyJwk": <span class="comment">...</span>
+      }, {
+        "id": <span class="comment">...</span>,
+        "type": <span class="comment">...</span>,
+        "controller": <span class="comment">...</span>,
+        "publicKeyMultibase": <span class="comment">...</span>
+      }]
+    }
+          </pre>
+
+          <p class="note"
+            title="Verification method controller(s) and controller(s)">
+The semantics of the `controller` property are the same when the
+subject of the relationship is the <a>controller document</a> as when the subject of
+the relationship is a <a>verification method</a>, such as a cryptographic public
+key. Since a key can't control itself, and the key controller cannot be inferred
+from the <a>controller document</a>, it is necessary to explicitly express the identity
+of the controller of the key. The difference is that the value of
+`controller` for a <a>verification method</a> is <em>not</em>
+necessarily a <a>controller</a>. <a>controllers</a> are expressed
+using the `<a>controller</a>` property at the highest level of the
+<a>controller document</a>.
+          </p>
+
+          <section>
+            <h3>Verification Material</h3>
+
+            <p>
+Verification material is any information that is used by a process that applies
+a <a>verification method</a>. The `type` of a <a>verification method</a> is
+expected to be used to determine its compatibility with such processes. Examples
+of verification methods include `JsonWebKey` and `Multikey`.
+A <a>cryptographic suite</a> specification is responsible for specifying the
+<a>verification method</a> `type` and its associated verification material
+format. For examples, see
+<a href="https://www.w3.org/TR/vc-di-ecdsa/">the Data Integrity ECDSA
+Cryptosuites</a> and <a href="https://w3c-ccg.github.io/lds-ed25519-2020/">
+the Data Integrity EdDSA Cryptosuites</a>. For a list of <a>verification
+method</a> types, please see the [[?SECURITY-VOCABULARY]].
+            </p>
+
+            <p>
+To increase the likelihood of interoperable implementations, this specification
```

Comment:
This sentence, basically says we couldn't agree... let's fix this, or not publish.